### DISCUSSION TITLE
pylance questions

### DISCUSSION THREAD

**Discussion by LiaoxyuCM** [microsoft/pylance-release#6588] [Q&A]
so how i import a module, pylance interrupt **"Cannot import this"**, pylance think this code didn't works. _The actually, it's works_

**Comment by rchiodo** [comment]
Generally, this means VS code doesn't have the correct interpreter selected. Pylance uses the interpreter you pick with this command here:


![image](https://github.com/user-attachments/assets/313a6439-03ee-46d5-898a-8f64523cb111)

For more information see the help here:
https://code.visualstudio.com/docs/python/environments#_manually-specify-an-interpreter
